dev/README.md

```diff
@@ -85,6 +85,45 @@ available.
 
 Ruby project dependency loading support coming soon.
 
+### Code Coverage
+
+#### Get Ready!
+in your oc_erchef window,
+
+```erlang
+(erchef@127.0.0.1)1> sync:pause().
+[info] Pausing Sync. Call sync:go() to restart
+
+ok
+```
+
+Then, for a specific module
+```erlang
+(erchef@127.0.0.1)2> cover:compile_beam(oc_chef_authz_acl).
+{ok,oc_chef_authz_acl}
+```
+
+or, for all the modules:
+
+```erlang
+Dirs = file:wildcard("/host/oc_erchef/apps/*/ebin") .
+[cover:compile_beam_dir(Dir) || Dir <- Dirs].
+```
+
+#### Run Pedant
+or whatever
```

Comment:
This feels a bit weird to have in a readme. Not quite sure what is being communicated here either.